tests/Queries.elm

```diff
@@ -206,21 +207,61 @@ testFirst output =
 testIndex : Single msg -> Test
 testIndex output =
     describe "Query.index"
-        [ describe "Query.index 0" <|
-            [ test "sees it's a <div class='container'>" <|
-                \() ->
+        [ describe "only 1 element"
+            [ fuzz (Fuzz.intRange -1 0) "matches" <|
```

Comment:
I'm not sure if there's any point in this being a fuzz test

Reply:
yes, your right, it's just less duplication and I was lazy. Will cleanup. I would love to have something like `testValues [-1, 0] ...` see my other comment.